src/components/incentives/IncentivesButton.tsx

```diff
@@ -3,9 +3,10 @@ import { ReserveIncentiveResponse } from '@aave/math-utils/dist/esm/formatters/i
 import { DotsHorizontalIcon } from '@heroicons/react/solid';
 import { Box, SvgIcon, Typography } from '@mui/material';
 import { useState } from 'react';
-import { useMeritIncentives, useUserMeritIncentives } from 'src/hooks/useMeritIncentives';
+import { useMeritIncentives } from 'src/hooks/useMeritIncentives';
 import { useRootStore } from 'src/store/root';
 import { DASHBOARD } from 'src/utils/mixPanelEvents';
+import { Side } from 'src/utils/utils';
```

Comment:
Lets use the `ProtocolAction` for this. 

```
import {

  ProtocolAction,
} from '@aave/contract-helpers';
ProtocolAction.borrow
```

Reply:
Ah good catch, I used the "Side" type that was implement on the "Superfest" tooltip PR.
I'll replace all Side by ProtocolAction, perfect